XCUITests/FxScreenGraph.swift

```diff
@@ -666,6 +682,8 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
     }
 
     map.addScreenState(ClearPrivateDataSettings) { screenState in
+        let table = app.tables.element(boundBy: 0)
+        screenState.tap(app.staticTexts["Website Data"], to: WebsiteDataSettings)
```

Comment:
Since it is almost done, lets try to add the accessibility identifier for Website Data menu, see [here](https://github.com/mrachamallu/firefox-ios/blob/b6372559e35a78bad35b06306f1b8348485d8c0b/Client/Frontend/Settings/ClearPrivateDataTableViewController.swift#L80)

Add that line, and then modify this line as:
screenState.tap(app.cells["WebsiteData"], to: WebsiteDataSettings)